<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTreeButton</class>
 <widget class="QPushButton" name="WidgetTreeButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>65</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget:hover { background-color: lightgray; border:none; }
</string>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pbArrow">
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed { background-color: none; border:none; }
QPushButton:checked { background-color: none; border:none; }</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/arrow_right</normaloff>
       <normalon>:/arrow_down</normalon>
       <disabledoff>:/empty</disabledoff>
       <disabledon>:/empty</disabledon>:/arrow_right</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelMainIcon">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/notebook</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelText">
     <property name="palette">
      <palette>
       <active/>
       <inactive/>
       <disabled/>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>36</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pbPlus">
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/add</normaloff>
       <disabledoff>:/empty</disabledoff>
       <disabledon>:/empty</disabledon>:/add</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbSearch">
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/search</normaloff>
       <disabledoff>:/empty</disabledoff>
       <disabledon>:/empty</disabledon>:/search</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
